### PR TITLE
Add MOONEVD space

### PR DESCRIPTION
**Pull request by jhlovesky**

### CODE REVIEW

**Review by huangtao163 — CHANGES_REQUESTED** [review]
add moonswap

**Comment by huangtao163** [comment]
add space moonswap

File: spaces/moonswap/skin.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.moonswap {
+  --primary-color: #384aff;
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```
Comment (e-asia) [inline spaces/moonswap/skin.scss:10]: Remove the skin.scss file to use the default theme.

Comment (huangtao163) [reply spaces/moonswap/skin.scss]: The update has been submitted, please preview

**Comment by huangtao163** [comment]
The update has been submitted, please preview

**Review by huangtao163 — CHANGES_REQUESTED** [review]
The update has been submitted, please preview

File: spaces/moonswap/index.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "defaultView": "core",
+  "address": "0xC65E282a82b5F0b02Dec20d32DCf12A89f5580bc",
+  "token": "0xC65E282a82b5F0b02Dec20d32DCf12A89f5580bc",
+  "core": [
+    "0xeF71a18bb044d98A861e7Ee9d6ff9262Fff32f12",
+    "0xfb8AC8edB1F67271DcceE572c904C9cA7d90a57E"
+  ],
+  "min": 0,
+  "invalid": []
+}
```
Comment (e-asia) [inline spaces/moonswap/index.json:14]: We have updated the format in `index.json`. 
Please add 2 new parameters `chainId` and `decimals`.
You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

**Review by e-asia — CHANGES_REQUESTED** [review]

**Review by huangtao163 — CHANGES_REQUESTED** [review]
parameters Chainid and decimals added

**Review by e-asia — APPROVED** [review]

**Pull request merged**